src/systests/test_logging.py

```diff
@@ -34,7 +34,7 @@
 from vmcontrol.vmmcontroller import VBoxController
 
 MAX_RUNTIME = 10 * 60  # Ten minutes
-pytestmark = [pytest.mark.systest]
+pytestmark = [pytest.mark.systest, pytest.mark.tmp_logging]
```

Comment:
Is this mark intentional or a dev leftover?

Reply:
Possibly a leftover 👀